src/catkin_pkg/package.py

```diff
@@ -137,6 +137,18 @@ def has_buildtool_depend_on_catkin(self):
         """
         return 'catkin' in [d.name for d in self.buildtool_depends]
 
+    def build_type(self):
+        """
+        Returns value of export/build_type element, or 'catkin' if unspecified.
+
+        :returns: package build type
+        :rtype: str
+        """
+        for e in self.exports:
+            if e.tagname == 'build_type':
+                return e.content
+        return 'catkin'
```

Comment:
Changed to `get_build_type`, added raise on multiple instances of the element. That's an increase in strictness from previously, although this is in the more freeform `<export>` section, and REP 127 doesn't actually have anything to say about `<build_type>`.

Maybe as part of https://github.com/ros-infrastructure/rep/pull/126 it should?